src/branding/default/branding.css

```diff
@@ -28,6 +28,10 @@ body.login-pf {
     font-weight: bold;
 }
 
+#brand:before {
+    content: "Cockpit";
```

Comment:
Where and when does this actually appear? On the standard web login page we actually don't want the product name to appear. I see that in default mode the page has class `hide-before`.

Reply:
This is the big banner that appears right above the login form.  I'll drop it again if that was a design decision.  It just seemed like a bug, when we have the OS name for all the other cases.

Reply:
Right, that was sort of my question -- I'm not saying that it's wrong, I'd just like to understand it (I'm not very familiar with the login CSS). *If* there is an OS name, then showing "Ubuntu 20.04" or "Fedora Linux" in #brand -- without a "Cockpit" prefix -- is desired. If there is no OS (such as in a bastion container or flatpak?) then showing "Cockpit" as a fallback makes sense, so that it's not completely empty.